optimum/neuron/utils/neuron_cc_wrapper

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/python3
```

Comment:
This assumes a system-wide install of Optimum Neuron, which isn't always the case.

Changing shebang to `#!/usr/bin/env python3` worked for my setup.